content/en/04-tutorials/01-pixie-101/07-kafka-monitoring.md

```diff
@@ -0,0 +1,182 @@
+---
+title: "Kafka Monitoring"
+metaTitle: "Tutorials | Pixie 101 | Kafka Monitoring"
+metaDescription: "Learn how to monitor Kafka using Pixie."
+order: 7
+---
+
+Debugging distributed messaging systems can be challenging. Pixie makes analyzing Kafka easier by using eBPF to [automatically capture](https://docs.px.dev/about-pixie/pixie-ebpf/) full-body Kafka requests without the need for manual instrumentation.
+
+This tutorial will demonstrate how to use Pixie to see:
+
+- [Topic-centric flow graph & topic summaries](#topic-centric-flow-graph)
+- [Kafka events with metadata](#kafka-events-with-metadata)
+- [Producer-consumer latency per topic and partition](#producer-consumer-latency)
+- [Consumer rebalancing duration](#consumer-rebalancing-duration)
+
+<YouTube youTubeId="42o5fURGXqI?t=1332"/>
+
+## Prerequisites
+
+1. You will need a Kubernetes cluster with Pixie installed. If you do not have a cluster, you can create a minikube cluster and install Pixie using one of our [install guides](https://docs.px.dev/installing-pixie/install-guides/).
+
+2. You will need to install the Kafka microservices demo application:
+
+> - Install the [Pixie CLI](/installing-pixie/install-schemes/cli/#1.-install-the-pixie-cli).
+> - Run `px demo deploy px-kafka` to install the demo app.
+> - Run `kubectl get pods -n px-kafka` to make sure all pods are ready before proceeding.
+> - Run `px demo interact px-kafka` and follow the directions to turn on the invoicing service delay.
+
+## Kafka Microservice Demo App
+
+The demo application you deployed in the [Prerequisites](#prerequisites) section has:
+
+- A single `order` topic.
+- One producer: the `order` service publishes messages to the `order` topic.
+- Two consumers: the `shipping` and `invoicing` services consume messages from the `order` topic. The `shipping` service is able to consume the order messages at the rate they are produced. The `invoicing` service is slower and unable to consume the order messages at the rate they are produced.
+
+::: div image-xl relative
+<svg title='' src='use-case-tutorials/kafka/kafka-microservice-app.png'/>
+:::
+
+To see the application's simple front end, navigate to the external IP (and port) of the `apache` service:
+
+```bash
+kubectl -n px-kafka get svc apache
+```
+
+To use the application, click on the Order page and scroll to the bottom of the page to select the `"Add Order"` button. Any orders added on the Order page should soon show up on both the Shipping and Invoicing pages.
```

Comment:
If the load gen is running, this may be hard to follow. Can we add a comment that the load gen is also generating orders so the reader is not confused.